src/common/engine.py

```diff
@@ -509,153 +510,169 @@ def _install_new_version(version: Version, download_url: str) -> bool:
         """Install the new stable version at the same directory level.
         Returns True on success, False otherwise."""
         new_version_dir: Path = Path('..') / f'sharly-chess-{version}'
-        if new_version_dir.exists():
-            logger.error(
-                'Version [%s] is already installed in directory [%s], please manually delete this folder before installing.',
-                version,
-                new_version_dir.absolute(),
-            )
-            return False
-        try:
-            with tempfile.TemporaryDirectory() as tmpdir:
-                tmp_dir: Path = Path(tmpdir)
-                logger.info(
-                    'Downloading release [%s] from GitHub ([%s])...',
-                    version,
-                    download_url,
-                )
-                response: Response = get(download_url, allow_redirects=True, timeout=10)
-                response.raise_for_status()
-                if not response:
-                    logger.error('No response from GitHub.')
-                    return False
-                if response.status_code != 200:
-                    logger.error(
-                        'Downloading failed with code [%d].', response.status_code
+        if not new_version_dir.exists():
```

Comment:
I think that we better keep this error.
```suggestion
        if new_version_dir.exists():
            logger.error(
                'Version [%s] is already installed in directory [%s], please manually delete this folder before installing.',
                version,
                new_version_dir.absolute(),
            )
            return False
        else:
```

Reply:
It's not an error, there's no point downloading it again, instead we fall through

Reply:
Not an error but at least a warning (I don't think that people not downloading the new release because of an already existing folder will understand it if they are not warned).